templates/security/fips.html

```diff
@@ -10,10 +10,10 @@
 <section class="p-strip--suru-topped">
  <div class="row u-equal-height u-vertically-center">
    <div class="col-8">
-     <h1>FIPS 140-2</h1>
-     <h2 class="p-heading--4">FIPS 140-2 validated cryptography for Linux workloads</h2>
-     <p>Developing and deploying open source workloads on regulated and high security environments can be challenging. There is a plethora of open source software following diverse development methods, and standards, but not always targeting a particular data protection standard. Canonical ensures that <a href="/public-cloud">Ubuntu Pro</a> and <a href="/advantage">Ubuntu Advantage</a> provide access to a set of core open source cryptographic packages that comply with the FIPS 140-2 requirements, the U.S. and Canada governments data protection standard.</p>
-     <p><a class="p-button--positive js-invoke-modal" href="/support/contact-us">Get Ubuntu with the FIPS140–2 validated modules</a></p>
+     <h1>FIPS 140</h1>
+     <h2 class="p-heading--4">FIPS 140 validated cryptography for Linux workloads</h2>
+     <p>Developing and running workloads for U.S. government regulated and high security environments requires a long and expensive validation process. Reduce your accreditation timeline and pass on your validation costs with the FIPS 140 certified cryptographic packages of <a href="/public-cloud">Ubuntu Pro</a> and <a href="/advantage">Ubuntu Advantage</a>.</p>
+     <p><a class="p-button--positive js-invoke-modal" href="/security/certifications#get-in-touch">Get Ubuntu with the FIPS140 validated modules</a></p>
```

Comment:
Just realised this with Pete, the point of this `href` is so if someone has javascript disabled they will still be able to see a contact form - but this is linking to another modal which doesn't make sense. So I would link to `/security/contact-us`

Reply:
Done